src/components/MapChart.vue

```diff
@@ -181,11 +178,11 @@ export default {
     updateData () {
       const self = this
 
-      const geolevel = this.selectedGeoLevel
-      const geocode = this.selectedGeoCode
-      const geoObject = this.getGeoObject(geolevel, geocode)
+      const selectedLevel = self.selectedGeoLevel
+      const selectedCode = self.selectedGeoCode
```

Comment:
Qu'est ce que représente self dans ce contexte ?

Reply:
Self = this dans ce context. C'est le component.